src/d3a/setup/grid_fees/constant_grid_fees.py

```diff
@@ -0,0 +1,59 @@
+"""
```

Comment:
i can not leave comment directly there, but `src/d3a/setup/grid_fees/non_compounded_grid_fees2.py`  is empty. Intentional?

Reply:
Because apparently it was empty in the master branch too! I will remove it, I blindly moved these 2 files in the new directory without looking at their content. Since it was empty in the first place, no reason to keep it.